Assets/VRTK/Scripts/Utilities/VRTK_SharedMethods.cs

```diff
@@ -180,6 +180,24 @@ public static float RoundFloat(float givenFloat, int decimalPlaces, bool rawFide
         }
 
         /// <summary>
+        /// The GetTypeUnknownAssembly method is used to find a Type without knowing the exact assembly it is in.
+        /// </summary>
+        /// <param name="typeName">The name of the type to get.</param>
+        /// <returns>The Type, or null if none is found.</returns>
+        public static Type GetTypeUnknownAssembly(string typeName)
+        {
+            var type = Type.GetType(typeName);
```

Comment:
Don't use `var` as you can't determine the variable type from the right of the equals